trunner/tools/phoenix.py

```diff
@@ -100,11 +123,7 @@ def _reader(self):
         self.proc.expect(pexpect.EOF, timeout=None)
 
     def _run(self):
-        try:
-            wait_for_dev(self.port, timeout=10)
-        except TimeoutError as exc:
-            raise PhoenixdError(f"couldn't find {self.port}") from exc
-
+        self.port = wait_for_vid_pid(0x16F9, 0x0003, timeout=10)
```

Comment:
- probably you should catch exception(s) and translate them to `PhoenixdError` to remain compatible with the current API
- `wait_for_dev` is now dead code, should be removed